src/data/java/com/github/minecraftschurlimods/arsmagicalegacy/data/AMPatchouliBookProvider.java

```diff
@@ -300,14 +298,21 @@ protected void addBooks(Consumer<BookBuilder<?, ?, ?>> consumer) {
+        TranslatedCategoryBuilder affinities = builder.addCategory("affinities", "Affinities", "", affinityHelper.getEssenceForAffinity(IAffinity.WATER))
+                .setSortnum(8);
+        affinities.addEntry("affinities", "Affinities", affinityHelper.getEssenceForAffinity(IAffinity.WATER))
+                .setPriority(true)
+                .addSimpleTextPage("Affinities are magical elements of sorts. All components (excluding some of the very powerful, reality-bending ones) have an affinity that is associated with them. That means that if you use spells with a certain affinity a lot, you will shift into that affinity.")
+                .addSimpleTextPage("Shifting into an affinity bears individual side effects, called abilities. Each affinity has different abilities. You can read about the abilities for each affinity in the dedicated chapters for them.$(br2)If you wish to see your current shift into an affinity, you can view your shifts in the Affinity tab of the $(l:blocks/occulus)Occulus$().")
+                .addSimpleTextPage("There is also an affinity essence for each affinity, which is used in intermediate crafting for spell parts associated with that affinity. Reports about lost affinity tomes have been spreading as well, though how to obtain or use them is currently subject to investigation.")
+                .build();
         var abilityRegistry = api.getAbilityRegistry();
         for (final IAffinity affinity : api.getAffinityRegistry()) {
             ResourceLocation id = affinity.getId();
             if (!id.getNamespace().equals(builder.getId().getNamespace()) || id.equals(IAffinity.NONE)) continue;
-            TranslatedEntryBuilder entry = affinities.addEntry(id.getPath(), affinity.getTranslationKey(), affinityHelper.getEssenceForAffinity(affinity))
-                                                     .addSimpleSpotlightPage(affinityHelper.getEssenceForAffinity(affinity), null, affinity.getTranslationKey())
-                                                     .addSimpleRecipePage("crafting", new ResourceLocation(id.getNamespace(), "affinity_essence_" + id.getPath()));
+            TranslatedEntryBuilder entry = affinities.addEntry(id.getPath(), affinity.getTranslationKey(), affinityHelper.getEssenceForAffinity(affinity));
+            entry.addSimpleTextPage(entry.getLangKey(0) + ".text");
+            entry.addSimpleRecipePage("crafting", new ResourceLocation(id.getNamespace(), "affinity_essence_" + id.getPath()));
```

Comment:
Why no spotlight? The recipe is temporary anyway and the highlight looks good

Reply:
My idea was to re-add the spotlight once the recipes were removed (bosses should be finished this week anyway)